pkg/grpc/packet/grpc_test.go

```diff
@@ -0,0 +1,87 @@
+package pktgrpc
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/mariomac/guara/pkg/test"
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/pbpacket"
```

Comment:
nit: import order

Reply:
done in https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/beef4dd954e19bba5aec090685e889b2ee2e233f